OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -503,8 +513,13 @@ private void loadData(@NonNull CourseComponent courseComponent) {
         }
         adapter.setData(courseComponent);
         if (adapter.hasCourseData()) {
+            setUpBulkDownloadHeader(courseComponent);
             errorNotification.hideError();
         } else {
+            if (isAdded()) {
```

Comment:
We should move the isAdded() function within utility function as well like
```java
public static void removeFragmentByTag(@NonNull Fragment parentFragment, @NonNull String tag) {
        if (parentFragment.isAdded()) {
            final FragmentManager fragmentManager = parentFragment.getChildFragmentManager();
            final Fragment fragment = fragmentManager.findFragmentByTag(tag);
            if (fragment != null) {
                fragmentManager.beginTransaction().remove(fragment)
                        .commit();
            }
        }
    }
```

Reply:
It might be possible that parent `Context` is an `activity` rather then `fragment`.

Reply:
We shall make a separate utility function for that, for now it make sense to consider fragment as a parent.

Reply:
Add appropriate java docs, specially explaining parentFragment